charts/latest/blob-csi-driver/templates/csi-blob-node.yaml

```diff
@@ -188,6 +225,17 @@ spec:
             {{- end }}
           resources: {{- toYaml .Values.node.resources.blob | nindent 12 }}
       volumes:
+{{- if .Values.node.enableBlobfuseProxy }}
+        - name: host-tmp
+          hostPath:
+            path: /tmp/
```

Comment:
the sidecar container also has /tmp dir, is it necessary to mount /tmp/ on agent node?

Reply:
This can be any Path in the host, since it is used to put blobfuse `deb` file and install it to the host. It is to keep the same as original [blobfuse-proxy](https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/deploy/blobfuse-proxy.yaml#L112-L115).

Reply:
then let's remove `/tmp`, it should work inside sidecar container

Reply:
We still need a hostPath to put blobfuse deb file in, a path only exists in sidecar container may not work.